### PR TITLE
Ports "Hovering your mouse over things now shows their name on the top of the screen"

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation

### Intent of your Pull Request

See below for original PR body.

In addition ports a back-end change to make /obj/screen into /atom/movable/screen, something something memory savings.

This should be test merged due to the above

### Why is this change good for the game?
Much more obvious way to see the name of stuff, instead of reading small text in the corner.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Pretty obvious

# Changelog

Original PR: https://github.com/tgstation/tgstation/pull/56729
feature I saw on other servers that I think is nice, because looking at the bottom left to see the name sucks

image
image
image

this is a pref, you can disable it.

:cl:  Mothblocks, Qustinnus
rscadd: Hovering your mouse over things now shows their name on the top of the screen
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
People no like.

**Comment by Bibby0110** [comment]
fuck

**Comment by adamsong** [comment]
> People no like.

Make it default to off then, I liked it

**Comment by Bibby0110** [comment]
> > People no like.
> 
> Make it default to off then, I liked it

That was the plan but merge conflicts
